templates/gallery/gallery-inner.amp.html

```diff
@@ -0,0 +1,207 @@
+          stroke="none"
+          fill="#F5F6FD"
+        ></path>
+      </svg>
+      {{> ./partials/hero.snip.html}}
+      <section class="gallery-base-section gallery-base-rich-text">
+        <div class="gallery-base-container mx-auto">
+          <div class="clearfix">
+            <div class="col col-12 md-col-9">
+              <p class="gallery-base-invert relative z1 mb2">
+                Vestibulum sit nibh. Et aliquam dolor. Sodales ac dui. Sed erat lacinia maecenas fusce accumsan vehicula aliquam nibh lorem dolor nulla. Facilisi quis tellus. Ut id sed. Porttitor habitant viverra proident scelerisque duis duis et turpis. Integer est condimentum. Suscipit nulla integer. Phasellus facilis vestibulum. Con pellentesque sed. Urna in risus sem fusce suscipit. In sed integer. Imperdiet mauris sit. Pretium vitae in. Fusce et vehicula. Nibh a aliquam. Proin massa posuere. Quis pulvinar vehicula.
+              </p>
+            </div>
+          </div>
+          <div class="gallery-inner-3d-row gallery-inner-perspective">
+            <div class="gallery-inner-3d-row-img-left">
+              <div class="gallery-inner-perspective">
+                <amp-img
+                  width="450"
+                  height="350"
+                  layout="responsive"
+                  src="../../img/gallery/stairs.jpg"
+                  srcset="
+                    ../../img/gallery/stairs.jpg 1x,
+                    ../../img/gallery/stairs@2x.jpg 2x
+                  "
+                  alt="pink stairs against a blue sky"
+                ></amp-img>
+              </div>
+            </div>
+            <div class="gallery-inner-3d-row-img-center">
+              <div class="gallery-inner-perspective">
+                <amp-img
+                  width="450"
+                  height="390"
+                  layout="responsive"
+                  src="../../img/gallery/plant.jpg"
+                  srcset="
+                    ../../img/gallery/plant.jpg 1x,
+                    ../../img/gallery/plant@2x.jpg 2x
+                  "
+                  alt="house plant on a chair on a yellow background"
+                ></amp-img>
+              </div>
+            </div>
+            <div class="gallery-inner-3d-row-img-right">
+              <div class="gallery-inner-perspective">
+                <amp-img
+                  width="500"
+                  height="600"
+                  layout="responsive"
+                  src="../../img/gallery/lamp.jpg"
+                  srcset="
+                  ../../img/gallery/lamp.jpg 1x,
+                  ../../img/gallery/lamp@2x.jpg 2x
+                  "
+                  alt="pink lamp on a blue wall"
+                ></amp-img>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="gallery-base-section gallery-base-rich-text">
+        <div class="gallery-base-container mx-auto">
+          <div class="clearfix">
```

Comment:
ditto

Reply:
Refactoring these to use flex layout is a significant amount of work.

All the responsive column will need to be recreated from scratch, since BassCSS doesn't have a flex equivalent of the `col col-<n>` system. This will also lead to a lot more custom CSS, since we'd need to essentially recreate the responsive grid system from scratch using flexbox.

If implemented we'd have classes like the one below all over the codebase.

```css
@media (--breakpoint-md) {
  .gallery-flex-md-col-9 {
    flex: 1 1 75%;
  }
}
```

Reply:
That is not correct - 

http://basscss.com/#basscss-flexbox
There is plenty of instances where col-* is used with flexboxes

infact you take a look a ampstart.com or some of our components we follow exactly this approach.

Reply:
Ah that's amazing! I had been looking at the v7 docs, my mistake -- I'll get this fixed ASAP.